examples/typegraphs/microservice-orchestration.py

```diff
@@ -25,17 +25,15 @@ def team_a(g: Graph):
     )
 
 
-# @typegraph(
-#     # skip:next-line
-#     cors=Cors(allow_origin=["https://metatype.dev", "http://localhost:3000"]),
-# )
-# def team_b(g: Graph):
-#     public = Policy.public()
-#
-#     deno = DenoRuntime()
-#
-#     g.expose(
-#         public,
-#         version=deno.static(t.integer(), 12),
-#         record=deno.static(t.struct({"weight": t.integer()}), {"weight": 100}),
-#     )
+@typegraph(
+    # skip:next-line
+    cors=Cors(allow_origin=["https://metatype.dev", "http://localhost:3000"]),
+)
+def team_b(g: Graph):
```

Comment:
The error on the test failure is different, but shouldn't the ts and python version of typegraph have the same name?

Reply:
I see them having the same name, can you point at where they differ?

Reply:
overlooked the second typegraph on my part, all looks good.